README.md

```diff
@@ -153,3 +153,24 @@ Setting up your local environment and database to run the functional tests again
   - Building your app locally will create a local image of it and give it the `:latest` tag.
   - Docker Compose will now use your local copy including your new stuff.
   - Then start the local container environment as described above.
+
+## Principles to follow when writing functional tests
+
+### Email addresses
+
+When using "dummy" email addresses in functional tests, try to stick to emails using one of the following domains:
```

Comment:
Do or do not - there is no `try`!

Reply:
I'm going to pretend not to get that reference...

Reply:
Translation: avoiding ambiguity in docs is a good thing 😸

Reply:
Well, ijust... don't want to be too much of a ball-breaker and realize that I may not have considered every circumstance with the 3 I've provided here.

Reply:
Fair enough!

Reply:
(because I'm likely to be the first to break the rules and am foreseeing my hypocrisy)

Reply:
I think @katstevens is right. It's good to be specific in our documentation. If we discover a new and justified thing that breaks the rules, then we can always update the docs. ;)

Reply:
changed